lib/std/mem.zig

```diff
@@ -3441,13 +3439,14 @@ test "indexOfMinMax" {
     try testing.expectEqual(indexOfMinMax(u8, "a"), .{ .index_min = 0, .index_max = 0 });
 }
 
+/// Swaps the two values.
```

Comment:
This doc comment is near useless. While this function is very self-documenting anyway, if you're going to add one, at least use clear language (i.e. referencing the actual parameters), e.g.: "Swaps the value pointed to by `a` with the value pointed to by `b`."

Reply:
Thanks, will change that.